docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+title: Horizen EON Explorer
+---
+
+
+
+The Horizen EON Explorer (Gobi Testnet Block Explorer) leverages the feature-rich Blockscout Explorer, an online tool for inspecting and analyzing the Gobi Testnet blockchain. Blockscout is an open-source code that allows the ability to customize a block explorer instance to view all transactions, past and current, on the blockchain. Customization can include information such as network hash rate and transaction details.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-landing.png").default} alt="landing-page" width="900" height="700" />
+</p>
+
+**Note:** Use **Search** to query for specific transaction, address (wallet or contract), block, and token.
+
+
+# Key Features
+
+The key features of the Gobi Testnet Block Explorer are modularized in each of the panels in the main landing page. 
+
+## Daily Transaction
+
+The **Daily Transaction** dashboard provides a real-time view of the blockchain with various data points.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-daily.png").default} alt="daily-transaction" width="900" height="700" />
+</p>
+
+The categories in the Daily Transaction dashboard includes:
+
+- A **graph view** of transactions processed during a time period. It includes metrics for tracking gas and transactions on a daily basis.
+- **Average block time** - This is the average time that a block is created.
+- **Total transactions** - The total number of transactions in a time period in the blockchain.
+- **Total blocks** - The total number of blocks in the blockchain.
+- **Wallet address** - The total number of wallet addresses associated with transactions in the blockchain.
+
+## Blocks
+
+The Block module provides key information about the block.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-blocks.png").default} alt="blocks-module" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Block number** - The number assigned to the block.
+- **Number of transactions** - The running number of transactions for the block.
+- **Number of bytes** - The amount of bytes for the block.
+- **Elapsed timer for the block** - Clock counting since block creation period.
+- **Miner (wallet) address** - The miner’s address. Click the address to Address Detail page.
+
+## Transactions
+
+The **Transaction** module provides key information about the transaction.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-transactions.png").default} alt="transactions" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **Address** - The wallet address associated with the transaction. 
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - The contract address associated with the transaction. Click the address to **Address Detail** page.

```

Comment:
This is similar to the comment I made above with the missing word. Also, if we are bolding "Address Detail" here, we may want to do it in the previous text as well?
```suggestion
- **Contract Address** - The contract address associated with the transaction. Click the address to navigate to the **Address Detail** page.

```

Reply:
correction made